src/ucp/api/ucp.h

```diff
@@ -266,7 +266,7 @@ enum ucp_ep_params_flags_field {
  * The enumeration is used to specify the behavior of @ref ucp_ep_close_nbx.
  */
 typedef enum {
-    UCP_EP_CLOSE_FLAG_FORCE         = 0, /**< @ref ucp_ep_close_nbx releases
+    UCP_EP_CLOSE_FLAG_FORCE = UCS_BIT(0) /**< @ref ucp_ep_close_nbx releases
```

Comment:
Seems like you changed UCP_EP_CLOSE_FLAG_FORCE from 0 to 1 which is ABI break.

Reply:
this flag was not part of previous release